vendor/github.com/zpatrick/fireball/router.go

```diff
@@ -65,7 +65,7 @@ func (r *BasicRouter) matchRoute(route *Route, req *http.Request) *RouteMatch {
 		return nil
 	}
 
-	pathVariables, ok := r.matchPathVariables(route, req.URL.Path)
+	pathVariables, ok := r.matchPathVariables(route, req.URL.RawPath)
```

Comment:
What happens in the event that fireball is updated and needs to be updated in d.ims.io?